README.md

```diff
@@ -625,6 +625,47 @@ The following settings are available per profile:
 * `stack.serverless.region` can be used to select the region to use when starting
   serverless projects.
 
+## Useful environment variables
```

Comment:
AFAIK there is no docs about all the environment variables that are defined in elastic-package. Just added this section to include all these references

WDYT ?